schemas/questions/types/general.json

```diff
@@ -35,9 +35,75 @@
         "type": "string",
         "enum": ["General"]
       },
+      "dynamic_answers": {
+        "type": "object",
+        "properties": {
```

Comment:
Need to make `values` and `answers` a mandatory property and need to disable additional properties.